### PR TITLE
Update application-services to version 63.0.0

### PR DESCRIPTION
**Pull request by mhammond**
The new app-services has a couple of fixes which would be great to get into fenix asap - https://github.com/mozilla/application-services/pull/3537 (crashes when autocompleting in some cases) and https://github.com/mozilla/application-services/pull/3567 (prevents bookmarks syncing)

### CODE REVIEW

File: components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/FirefoxAccount.kt [hunk 1]
```diff
@@ -105,16 +106,24 @@ class FirefoxAccount internal constructor(
     }
 
     override fun beginOAuthFlowAsync(scopes: Set<String>) = scope.async {
+        // Eventually we should specify this as a param here, but for now, let's
+        // use a generic value (it's used only for server-side telemetry, so the
+        // actual value doesn't matter much)
+        val entryPoint = "android-components"
```
Comment (rfk) [inline components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/FirefoxAccount.kt:112]: nit: could we add this as an optional param in `beginOAuthFlowSync`, so we can update consumers to pass in their own values without waiting on another release of a-c?

Comment (mhammond) [reply components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/FirefoxAccount.kt:112]: good idea!

**Comment by rfk** [comment]
@mhammond looks like the build is failing here with some style errors:

```
[task 2020-09-11T04:10:02.082Z] /builds/worker/checkouts/src/components/concept/sync/src/main/java/mozilla/components/concept/sync/OAuthAccount.kt - 5min debt
[task 2020-09-11T04:10:02.082Z] 	MaxLineLength - [beginPairingFlowAsync] at /builds/worker/checkouts/src/components/concept/sync/src/main/java/mozilla/components/concept/sync/OAuthAccount.kt:83:5
[task 2020-09-11T04:10:02.082Z] 
[task 2020-09-11T04:10:02.082Z] Overall debt: 5min
[task 2020-09-11T04:10:02.082Z] 
[task 2020-09-11T04:10:02.082Z] style - 5min debt
[task 2020-09-11T04:10:02.082Z] 	MaxLineLength - [beginPairingFlowAsync] at /builds/worker/checkouts/src/components/concept/sync/src/main/java/mozilla/components/concept/sync/OAuthAccount.kt:83:5
[task 2020-09-11T04:10:02.082Z] 
```

**Comment by jonalmeida** [comment]
bors try

I fixed the PR so we can run our basic UI tests for sync on it.

**Comment by mhammond** [comment]
Sorry, I didn't get emails for the above messages until the one from bors 1/2 an hour ago :( Would you like me to look into the current errors?

**Comment by jonalmeida** [comment]
Sure! It looks like we just need to fix the tests of the matcher in the test to expect the new arg.

**Comment by jonalmeida** [comment]
bors try

I filed https://github.com/mozilla-mobile/android-components/issues/8415 for the intermittent test failure. Let's run try again!

**Comment by mhammond** [comment]
@jonalmeida is this ready to merge?

**Comment by jonalmeida** [comment]
I did some manual testing and noticed that Send Tab and receiving a tab doesn't seem to be working (even with manually clicking Sync Now).

Although, it looks like this is an issue that is already on today's Fenix Nightly (AS 61.0.13) when tested against desktop: 82.0a1 (2020-09-14) (64-bit), 79.0 (64-bit), and 80.0.1 (64-bit)

I'm guessing this is an issue on the service side in that case?

```
mozilla.appservices.fxaclient.FxaException$Unspecified: Remote server error: '500' '203' 'Internal Server Error' 'A backend service request failed.'
```

**Review by jonalmeida — APPROVED** [review]
I think we should be fine with landing this based on a manual smoke test. I imagine the send tab server-side will be fixed soon since it probably affects all platforms.

**Comment by jonalmeida** [comment]
Send Tab is now working in both directions. 👌

**Pull request merged**